script/api.py

```diff
@@ -0,0 +1,64 @@
+import os
+
+import psycopg2
+from flask import Flask, jsonify, request, session
+
+app = Flask(__name__)
+
+hostname = os.getenv('POSTGRES_HOST')
+database = os.getenv('POSTGRES_DB')
+username = os.getenv('POSTGRES_USER')
+
+port_id = 5432
+conn = psycopg2.connect(
+    host=hostname,
+    dbname=database,
+    user=username,
+    port=port_id
+)
+
+cur = conn.cursor()
+
+
+@app.route('/login', methods=['POST'])
+def login():
+
+    email = request.json.get('email')
+    password = request.json.get('password')
+    cur.execute("SELECT * FROM users WHERE email='{}' ".format(email))
+    user_email = cur.fetchone()
+    
+    if user_email :
+        cur.execute(
+            "SELECT * FROM users WHERE email='{}' AND password=crypt('{}',password)".format(email, password))
+        user = cur.fetchone()
+        if user:
+            print(user)
+            id = user[0]
+            if 'user_id' in session:
+                return jsonify({"username": user[1]})
+            else:
+
+                session['user_id'] = id
+                return jsonify({"username": user[1]})
+        else:
+            return jsonify({"message": "The password that you've entered is incorrect. "}),401
+
+    else:
+        return jsonify({"message": "E-mail address does not exist."}),401
```

Comment:
Ideally its a bad practice to separately tell this error to user at login time. We should simply say "Either email or password is incorrect"